packages/llvm/llvm.3.9/files/install.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+make=$1
+libdir=$2
+
+# Create build dir
+mkdir build
+cd build
+
+# Configure
+cmake ..
+#    -DLLVM_BUILD_LLVM_DYLIB:BOOL=ON \
+#    -DLLVM_LINK_LLVM_DYLIB:BOOL=ON \
+#    ..
+
+# Build
+grep "^ocaml_llvm.*/fast:$" Makefile | tr ":\n" " " | xargs "$make"
```

Comment:
Thanks. I updated the installation and wrote some patches, what do you think ? (I removed the script, I think it's better like this).
I'll try to ask on #llvm about the linking problem.